python/pyproject.toml

```diff
@@ -121,3 +129,20 @@ show_error_context = true
 testpaths = [
     "pytests",
 ]
+
+[tool.semantic_release]
```

Comment:
With this configuration, the `python-semantic-release` tool can be used to generate version numbers, tag branches, and push changelogs to GitHub. Need to verify / test. It will use the `-a` prerelease token for the `main` branch, allowing us to (I hope) do continuous deployment using auto-incrementing alpha versions off main.